modules/ztp-sno-siteconfig-config-reference.adoc

```diff
@@ -0,0 +1,82 @@
+|Description
+
+|`spec.cpuPartitioningMode`
+|Configure workload partitioning by setting the value for `cpuPartitioningMode` to `AllNodes`.
+To complete the configuration, specify the `isolated` and `reserved` CPUs in the `PerformanceProfile` CR.
+You enable workload partitioning during cluster installation only.
+
+|`metadata.name`
+|Set `name` to `assisted-deployment-pull-secret` and create the `assisted-deployment-pull-secret` CR in the same namespace as the `SiteConfig` CR.
+
+|`clusterImageSetNameRef`
+|Configure the image set available on the hub cluster.
+To see the list of supported versions on your hub cluster, run `oc get clusterimagesets`.
+
+|`installConfigOverrides`
+a|Set the `installConfigOverrides` field to enable or disable optional components prior to cluster installation.
+[NOTE]
+====
+`NodeTuning` is required for 4.13 and later.
+====
+
+|`spec.clusters.clusterLabels`
+|Configure cluster labels to correspond to the `bindingRules` field in the `PolicyGenTemplate` CRs that you define.
+For example, `policygentemplates/common-ranGen.yaml` applies to all clusters with `common: true` set, `policygentemplates/group-du-sno-ranGen.yaml` applies to all clusters with `group-du-sno: ""` set.
+
+|`spec.clusters.crTemplates.KlusterletAddonConfig`
+|Optional. Set `KlusterletAddonConfig` to `KlusterletAddonConfigOverride.yaml to override the default `KlusterletAddonConfig` that is created for the cluster.
+
+|`spec.clusters.nodes.hostName`
+|For single-node deployments, define a single host.
+For three-node deployments, define three hosts.
+For standard deployments, define three hosts with `role: master` and two or more hosts defined with `role: worker`.
+
+|`spec.clusters.nodes.bmcAddress`
+|BMC address that you use to access the host. Applies to all cluster types. {ztp} supports iPXE and virtual media booting by using Redfish or IPMI protocols. To use iPXE booting, you must use {rh-rhacm} 2.8 or later. For more information about BMC addressing, see the "Additional resources" section.
+
+|`spec.clusters.nodes.biosConfigRef`
+|Optional. Configure required host BIOS settings.
+
+|`spec.clusters.nodes.bmcAddress`
+|BMC address that you use to access the host.
+Applies to all cluster types.
+{ztp} supports iPXE and virtual media booting by using Redfish or IPMI protocols.
+To use iPXE booting, you must use {rh-rhacm} 2.8 or later.
+For more information about BMC addressing, see the "Additional resources" section.
+
+|`spec.clusters.nodes.bmcCredentialsName`
+|Configure the `bmh-secret` CR that you separately create with the host BMC credentials.
+When creating the `bmh-secret` CR, use the same namespace as the `SiteConfig` CR that provisions the host.
+
+|`spec.clusters.nodes.bootMode`
+|Set the boot mode for the host to `UEFI`.
+The default value is `UEFI`. Use `UEFISecureBoot` to enable secure boot on the host.
+
+|`spec.clusters.nodes.diskPartition`
+|Optional. The provided example `diskPartition` is used for the image registry.
```

Comment:
Prefer to not reference an image registry here. This can be simply "is used to configure additional disk partitions"